### ISSUE TITLE
ModelRun passes directly to COMPLETE after creation

### ISSUE THREAD

**Issue by 1Mouaci** [t-wolfeadam/Alpyne#20]
```
[2022-08-08 18:50:58] INFO    @ com.anylogic.alpyne.AlpyneServer main 
                              | [PID=13756] Initializing @ 'C:\Users\h40182\Downloads\systeme distribution' with arguments: {path=., port=51150, dev_mode=false}
[2022-08-08 18:50:58] FINER   @ ModelManager.<init>
                              | ENTRY {0}	.
[2022-08-08 18:50:58] FINER   @ BonsaiBrainLearningPlatform.<init>
                              | ENTRY {0}	{}
[2022-08-08 18:50:58] INFO    @ Version.load
                              | Loading from experiment
[2022-08-08 18:51:00] INFO    @ Version.load
                              | Inputs      = [(DOUBLE){START_TIME}=0.0, (DATE_TIME){START_DATE}=Tue May 17 00:00:00 CEST 2022, (DOUBLE){STOP_TIME}=Infinity, (DATE_TIME){STOP_DATE}=null, (LONG){RANDOM_SEED}=null]
                                Outputs     = [(DOUBLE)score=null]
                                Config      = [(DOUBLE)config=null, (DOUBLE){START_TIME}=0.0, (DATE_TIME){START_DATE}=Tue May 17 00:00:00 CEST 2022, (DOUBLE){STOP_TIME}=Infinity, (DATE_TIME){STOP_DATE}=null, (LONG){RANDOM_SEED}=null]
                                Observation = [(DOUBLE)lignes_tr=null]
                                Action      = [(INTEGER)local_v1=null, (INTEGER)local_v2=null, (INTEGER)local_v3=null]
[2022-08-08 18:51:00] WARNING @ org.glassfish.jersey.server.wadl.WadlFeature configure 
                              | JAX-B API not found . WADL feature is disabled.
[2022-08-08 18:51:00] INFO    @ org.glassfish.grizzly.http.server.NetworkListener start 
                              | Started listener bound to [localhost:51150]
[2022-08-08 18:51:00] INFO    @ org.glassfish.grizzly.http.server.HttpServer start 
                              | [HttpServer] Started.
[2022-08-08 18:51:02] FINER   @ VersionsResource.getVersion
                              | ENTRY
[2022-08-08 18:51:07] FINE    @ RunsResource.<init>
                              | Initializing Runs Resource endpoint
[2022-08-08 18:51:08] FINER   @ RunsResource.createNewRun
                              | ENTRY {0}	{"experimentType": "REINFORCEMENT_LEARNING", "inputs": [{"name": "config", "type": "DOUBLE", "value": 300, "units": null}, {"name": "{START_TIME}", "type": "DOUBLE", "value": 0, "units": "SECOND"}, {"name": "{START_DATE}", "type": "DATE_TIME", "value": "2022-05-16T22:00:00Z", "units": null}, {"name": "{STOP_TIME}", "type": "DOUBLE", "value": Infinity, "units": "SECOND"}, {"name": "{STOP_DATE}", "type": "DATE_TIME", "value": null, "units": null}, {"name": "{RANDOM_SEED}", "type": "LONG", "value": 1, "units": null}]}
[2022-08-08 18:51:08] INFO    @ ModelRunPlatform.<init>
                              | Initializing ModelRun 1
[2022-08-08 18:51:08] FINER   @ ModelRunPlatform.lockUntilServerRequests
                              | ENTRY
[2022-08-08 18:51:08] FINER   @ ModelRunPlatform.run
                              | ENTRY {0}	1
[2022-08-08 18:51:08] FINER   @ BonsaiBrainLearningPlatform.<init>
                              | ENTRY {0}	{}
[2022-08-08 18:51:08] FINER   @ BonsaiBrainLearningPlatform.<init>
                              | ENTRY {0}	{"id": 1}
[2022-08-08 18:51:08] INFO    @ BonsaiBrainLearningPlatform.run
                              | Platform beginning episode 1
[2022-08-08 18:51:08] FINER   @ BonsaiBrainLearningPlatform.serverSetupNewEpisode
                              | Platform now requesting configuration
[2022-08-08 18:51:08] FINER   @ ModelRunPlatform.lockUntilServerRequests
                              | RETURN
[2022-08-08 18:51:08] FINER   @ ModelRunPlatform.setConfiguration
                              | ENTRY {0}	[(DOUBLE)config=300, (DOUBLE){START_TIME}=0, (DATE_TIME){START_DATE}=2022-05-16T22:00:00Z, (DOUBLE){STOP_TIME}=Infinity, (DATE_TIME){STOP_DATE}=null, (LONG){RANDOM_SEED}=1]
[2022-08-08 18:51:08] FINER   @ ModelRunPlatform.setConfiguration
                              | Platform ready to accept inputs
[2022-08-08 18:51:08] FINER   @ BonsaiBrainLearningPlatform.clientStartNewEpisode
                              | Client notifying platform of sent configuration
[2022-08-08 18:51:08] FINER   @ ModelRunPlatform.getRunState
                              | ENTRY
[2022-08-08 18:51:08] FINER   @ BonsaiBrainLearningPlatform.serverSetupNewEpisode
                              | Platform received configuration
[2022-08-08 18:51:08] INFO    @ ModelRunPlatform.getRunState
                              | [eps 1, step 0] stage: START | fin deets: (null,null)
[2022-08-08 18:51:08] FINER   @ RunsResource.createNewRun
                              | RETURN {0}	OutboundJaxrsResponse{status=201, reason=Created, hasEntity=true, closed=false, buffered=false}
1.0
[2022-08-08 18:51:08] INFO    @ BonsaiBrainLearningPlatform.run
                              | Platform beginning step 1
[2022-08-08 18:51:08] INFO    @ BonsaiBrainLearningPlatform.run
                              | Terminal due to STOP_TIMEDATE (null); ending episode
[2022-08-08 18:51:08] INFO    @ BonsaiBrainLearningPlatform.run
                              | Platform finished episode 1
[2022-08-08 18:51:08] FINER   @ BonsaiBrainLearningPlatform.serverFinishEpisode
                              | Platform updated observation; now requesting reset
[2022-08-08 18:51:19] FINER   @ RunsResource.getRunProgress
                              | ENTRY {0}	1
[2022-08-08 18:51:19] FINER   @ ModelRunPlatform.getRunState
                              | ENTRY
[2022-08-08 18:51:19] INFO    @ ModelRunPlatform.getRunState
                              | [eps 1, step 1] stage: FINISH | fin deets: (STOP_TIMEDATE,null)
[2022-08-08 18:51:29] FINER   @ RunsResource.executeRLCommand
                              | ENTRY {0} {1}	1, {"command": "ACTION", "argument": [{"name": "local_v1", "type": "INTEGER", "value": 0, "units": null}, {"name": "local_v2", "type": "INTEGER", "value": 0, "units": null}, {"name": "local_v3", "type": "INTEGER", "value": 0, "units": null}]}
[2022-08-08 18:51:29] INFO    @ RunsResource.executeRLCommand
                              | For run 1 issuing command ACTION with argument [(INTEGER)local_v1=0, (INTEGER)local_v2=0, (INTEGER)local_v3=0]
[2022-08-08 18:51:29] FINER   @ ModelRunPlatform.setAction
                              | ENTRY {0}	[(INTEGER)local_v1=0, (INTEGER)local_v2=0, (INTEGER)local_v3=0]
[2022-08-08 18:51:29] FINER   @ ModelRunPlatform.getRunState
                              | ENTRY
[2022-08-08 18:51:29] INFO    @ ModelRunPlatform.getRunState
                              | [eps 1, step 1] stage: FINISH | fin deets: (STOP_TIMEDATE,null)
[2022-08-08 18:51:29] SEVERE  @ ModelManager.executeRequest
                              | Problem executing request
                                java.lang.IllegalStateException: Platform not in correct stage: (FINISH, true)
                                	at com.anylogic.alpyne.sim.api.ModelRunPlatform.setAction(ModelRunPlatform.java:104)
                                	at com.anylogic.alpyne.sim.ModelManager.executeRequest(ModelManager.java:114)
                                	at com.anylogic.alpyne.server.api.RunsResource.executeRLCommand(RunsResource.java:240)
                                	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
                                	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
                                	at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52)
                                	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124)
                                	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167)
                                	at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$ResponseOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:176)
                                	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79)
                                	at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:475)
                                	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:397)
                                	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:81)
                                	at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:255)
                                	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)
                                	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)
                                	at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
                                	at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
                                	at org.glassfish.jersey.internal.Errors.process(Errors.java:244)
                                	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265)
                                	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:234)
                                	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:680)
                                	at org.glassfish.jersey.grizzly2.httpserver.GrizzlyHttpContainer.service(GrizzlyHttpContainer.java:356)
                                	at org.glassfish.grizzly.http.server.HttpHandler$1.run(HttpHandler.java:190)
                                	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:535)
                                	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:515)
                                	at java.base/java.lang.Thread.run(Thread.java:833)
                                
```
The model Run passes to the COMPLETE status with the reason of END_DATETIME even if i made sure that the model time in anylogic was never.
i tried to figure out the reason but i couldn't.

**Comment by mjhoover1** [comment]
Try setting the configuration client.configuration_template.engine_stop_time before client.create_reinforcement_learning(cfg)

**Comment by Haohao990828** [comment]
I have encountered the same problem as you.   Could you please tell me if you have solved it and how did you do it?    
Thank you！

**Comment by t-wolfeadam** [comment]
Closing as this is likely resolved in v1.0.0. Feel free to create a new issue if it remains.
